_sass/1-modules/_mixins.scss

```diff
@@ -0,0 +1,84 @@
+// SASS mixins =================================================================
+
+@mixin box-shadow($shadow) {
+  box-shadow: $shadow;
+}
+
+@mixin border-radius($radius) {
+  border-radius: $radius;
+  background-clip: padding-box;
+}
+
```

Comment:
AFAIK this is unused in the codebase and can be removed! A holdover from the base styles.

Reply:
yeah let me see if there are mixins not in use. but I'll keep my eye out to be sure this isn't active.